pyproject.toml

```diff
@@ -11,8 +11,8 @@ keywords = []
 packages = [ { include = "historian", from = "src" }, { include = "historian", from = "tests" } ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"
-volttron = ">=10.0.2rc0,<11.0"
+python = ">=3.10,<4.0"
+volttron = ">=10.0.3a9,<11.0"
```

Comment:
This will be 10.0.4rc0 soon as the thing is released.

Reply:
won't poetry automatically pull the latest version that is within the given version range?

Reply:
yes...wondering if we want to be explicit though....I did notice that with the modbustk lib I have to update the base driver to volttron-testing 0.4 in order for it to be updated.